tools/ci_build/github/azure-pipelines/post-merge-jobs.yml

```diff
@@ -1,34 +1,6 @@
-# these parameters are provided to selectively enable build configurations when running manually
-# all build configurations should be enabled by default
-
-parameters:
-- name: EnableMimalloc
-  displayName: Enable Mimalloc build?
-  type: boolean
-  default: true
-
-- name: EnableNoAbseil
-  displayName: Enable no Abseil build?
-  type: boolean
-  default: true
-
-- name: EnableMinimalWithNoExceptions
-  displayName: Enable minimal build with no exceptions?
-  type: boolean
-  default: true
-
-- name: EnableDebugNodeIo
-  displayName: Enable build with debug node I/O?
-  type: boolean
-  default: true
-
-- name: EnableCodeCoverage
-  displayName: Enable code coverage build?
-  type: boolean
-  default: true
-
-jobs:
-- ${{ if parameters.EnableMimalloc }}:
+stages:
+- stage: Mimalloc
+  jobs:
```

Comment:
Add 
```
dependsOn: []
```
To each of the stages. 

Like: https://github.com/microsoft/onnxruntime/blob/main/tools/ci_build/github/azure-pipelines/win-ci-pipeline.yml#L9